src/RpcClient.ts

```diff
@@ -1060,4 +1062,48 @@ export class RpcClient {
       throw new Error(`Error in getTokenAccounts: ${error}`);
     }
   }
-}
+
+  /**
+   * Send a transaction
+   * @param {Transaction} transaction - The transaction to send
+   * @param {HeliusSendOptions} options - Options for sending the transaction
+   * @returns {Promise<TransactionSignature>} - The transaction signature
+   */
+  async sendTransaction(
+    transaction: Transaction | VersionedTransaction,
+    options: HeliusSendOptions = {
+      skipPreflight: true,
+    }
+  ): Promise<TransactionSignature> {
+    let rawTransaction: string;
+    if (transaction instanceof VersionedTransaction) {
```

Comment:
Why do we need to do this?

Reply:
in web3js, `sendTransaction` has an overloaded version that accepts a `VersionedTransaction`, and a deprecated version that accepts `Transaction`. I wanted to make our version more user-friendly to anyone who is still using the deprecated `Transaction` interface. I can remove support for `Transaction` and only leave `VersionedTransaction` if you prefer